app/views/users_mailer/credentials.html.slim

```diff
@@ -0,0 +1,12 @@
+div
+  h1 Your credentials
+div
+  h3
+    = "#{@greeting}, #{@user.full_name}."
```

Comment:
why you have here @greeting? Just put Hello in template
never repeat this error